wca-guidelines.md

```diff
@@ -183,6 +183,7 @@ To be more informative, each Guideline is classified using one of the following
 - A3c4++) [CLARIFICATION] The competitor may remove the object during the solve, if they did not do it during the inspection (e.g. they forgot).
 - A5b+) [CLARIFICATION] While inspecting or solving the puzzle, the competitor may touch the puzzle with any part of their body. Exception: 3x3x3 One-Handed (see [Regulation C1b](regulations:regulation:C1b)).
 - A5b++) [CLARIFICATION] If a part of the competitor's puzzle pops out (see [Regulation 5a](regulations:regulation:5a)), the competitor must not ask for assistance retrieving or locating the part and the judge should attempt to prevent others from assisting. If someone picks up a popped part for the competitor, the judge or competitor may ask them to place it on the solving station and not to assist any further.
+- A5b+++) [CLARIFICATION] If someone picks up a popped part of the competitor, the WCA Delegate should not grant the competitor an extra attempt unless a separate incident occurs.
```

Comment:
```suggestion
- A5b++) [CLARIFICATION] If a part of the competitor's puzzle is popped out or detached (see [Regulation 5a](regulations:regulation:5a)), the competitor must not ask for assistance retrieving or locating the part and the judge should attempt to prevent others from assisting. If someone picks up a popped part for the competitor, the judge or competitor may ask them to place it on the solving station and not to assist any further.
- A5b+++) [CLARIFICATION] If someone picks up a popped or detached part of the competitor, the WCA Delegate should not grant the competitor an extra attempt unless a separate incident occurs.
```

Reply:
```suggestion
- A5b++) [CLARIFICATION] If a part of the competitor's puzzle is popped out or detached (see [Regulation 5a](regulations:regulation:5a)), the competitor must not ask for assistance retrieving or locating the part and the judge should attempt to prevent others from assisting. If someone picks up a popped part for the competitor, the judge or competitor may ask them to place it on the solving station and not to assist any further.
- A5b+++) [CLARIFICATION] If someone picks up a popped or detached part of the competitor's puzzle, the WCA Delegate should not grant the competitor an extra attempt unless a separate incident occurs.
```
One further unconnected suggest for something that read a little oddly when looking at this again. The part belongs to the competitor but is not a physical part of them!

Reply:
Oops, yeah, we noticed it but I forgot to include it into the suggestion. Thanks Oliver!